crates/block-producer/src/test_utils/block.rs

```diff
@@ -169,6 +169,7 @@ pub(crate) fn note_created_smt_from_note_batches<'a>(
     BlockNoteTree::with_entries(note_leaf_iterator).unwrap()
 }
 
+#[cfg(test)]
```

Comment:
Why is this needed here? I thought this entire module is enabled only for tests.